spec/system/consumer/multilingual_spec.rb

```diff
@@ -27,7 +27,7 @@
       visit root_path
       expect(get_i18n_locale).to eq 'en'
       expect(get_i18n_translation('label_shops')).to eq 'Shops'
-      expect(cookies).to be_empty
+      expect(cookies_name).not_to include('locale')
```

Comment:
Interesting. So I guess that we are starting a session earlier now for tracking timestamps?

Reply:
I think that's what's happening, but I tried disabling the timestamp check but still ran in the issue. Anyway, the cookie assertion was wrong in my opinion